util/src/main/java/tc/oc/pgm/util/nms/reflect/Refl.java

```diff
@@ -157,9 +193,13 @@ interface Block {
     Object getBlockData(Object self);
 
     @Reflect.Method("q")
+    @Reflect.Method(value = "q", parameters = IBlockData.class)
```

Comment:
are these two redundant?

Reply:
So it turns out that "q" was incorrect. I didn't catch this because this method was never called due to a bug in blockdrops meaning that `wrong-tool` would never apply. I've fixed that and tested it in sportpaper, 1.9, and 1.12